obspy/io/quakeml/core.py

```diff
@@ -181,6 +181,17 @@ def _xpath2obj(self, xpath, element=None, convert_to=str, namespace=None):
             warnings.warn(msg % (text, convert_to))
         return None
 
+    def _set_enum(self, xpath, element, obj, key):
+        obj_type = obj._property_dict[key]
+        if not isinstance(obj_type, Enum):  # pragma: no cover
+            raise ValueError
+        value = self._xpath2obj(xpath, element)
+        try:
+            setattr(obj, key, value)
+        except ValueError as e:
+            msg = "%s. Will be ignored." % e.args[0]
```

Comment:
Will be ignored -> 'The attribute %s will not be set and will be missing in the resulting object' % key ?